extension/src/popup/components/manageAssets/TrustlineError/index.tsx

```diff
@@ -39,17 +39,19 @@ const renderError = (
     case TRUSTLINE_ERROR_STATES.NOT_ENOUGH_LUMENS:
       return (
         <InfoBlock variant={InfoBlock.variant.error}>
-          <p className="TrustlineError__title">Not enough lumens</p>
-          <p>0.500001 XLM are required to add a new asset.</p>
-          <p className="TrustlineError__links">
-            <Link to="https://developers.stellar.org/docs/glossary/minimum-balance/#changes-to-transaction-fees-and-minimum-balances">
-              Learn more about transaction fees
-            </Link>
-            <br />
-            <Link to="https://developers.stellar.org/docs/glossary/accounts/#liabilities">
-              Learn more about account reserves
-            </Link>
-          </p>
+          <div>
```

Comment:
quick note with the new InfoBlock. It accepts only one child. This makes sure the icon always lines up.

Typescript will enforce this, so no need to commit to memory. Just wanted to explain why I'm wrapping this in a div